frontend/src/App.js

```diff
@@ -61,7 +66,7 @@ function App({ themeToggler }) {
   const MyRoute = Route;
 
   // Render spinner if page loading
-  if (loading) {
+  if (loading || user.year === 2025) {
```

Comment:
Be warned: Yale is not currently listing years for 2025 students in the directory. They simply have no year listed, and unfortunately there's no way on the Yalies side to distinguish them from other people with no year listed.

Reply:
Yep @kevinhu mentioned that. Since we're deploying the beta in june as the main version, it may not even matter. just thought I should include it in the case that Yale lists their year by June